### DISCUSSION TITLE
혹시 PR리뷰 때 리뷰어들이 피드백 준 것을 업데이트 하는 과정도 거치나요?

### DISCUSSION THREAD

**Discussion by chadseok** [wanted-pre-onboarding-side-project/habit-tracker#41] [Q&A] [answered]
실시간으로 피드백 되어 변경된 코드를 올리는 것? 도 하실 건지 궁금합니다

**Comment by jiyeon2** [comment, marked as answer]
pr 리뷰가 단순히 코드에 대한 질문응답 과정이 아닌
upstream 브랜치에 push 할 코드를 검사하는 절차라고 생각하고

main에 merge 되면 프로젝트에 반영될 코드이기 때문에 
저는 업데이트를 진행했으면 합니다~

@chadseok @greyHairChooseLife 님은 어떻게 생각하시나요?

참고로 제가 생각했던 리뷰 과정은 이렇습니다
1. 1차 피드백 받고 나서 리뷰어의 코멘트에 답변하기, 피드백 바탕으로 코드 수정 적용
2. 2차 피드백 받기

**Reply by greyHairChooseLife** [reply]
저도 지연님 생각에 100% 동의합니다!
